src/quickpicks/aiModelPicker.ts

```diff
@@ -33,6 +34,16 @@ export async function showAIProviderPicker(
 	container: Container,
 	current: AIModelDescriptor | undefined,
 ): Promise<ProviderQuickPickItem | undefined> {
+	if (!configuration.get('ai.enabled')) {
+		await window.showQuickPick([{ label: 'OK' }], {
+			title: 'AI is Disabled',
+			placeHolder: 'GitLens AI features have been disabled via settings',
+			canPickMany: false,
+		});
+
+		return undefined;
+	}
+
```

Comment:
Please move this into a helper shared with the below. And it would be create to have an entry to re-enable the setting.

Reply:
Done